src/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- displays site properly based on user's device -->
+
+  <link rel="icon" type="image/png" sizes="32x32" href="./images/logo-sm.png">
+
+  <title>Clara Store</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Barlow+Semi+Condensed:wght@500;600&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+  <header class="wrapper">
+    <nav class="navbar p-200">
+      <a href="#">
+        <img width="200" height="70" src="./images/logo-lg.png"
+          alt="logo contains a butterfly and company name(Clara Store)"></a>
+      <div class="nav-list">
+        <ul class="nav-menu">
+          <img class="close-menu p-300" src="./images/icon-close.svg" alt="close menu icon (cross)">
+          <li><a class="list-item text-dark-grayish-blue" data-content="products" href="javascript:void(0)">Products (<span id="pCount"></span>)</a></li>
+          <li><a class="list-item text-dark-grayish-blue" data-content="contact" href="javascript:void(0)">Contact</a></li>
+          <li><a class="list-item text-dark-grayish-blue" data-content="about" href="javascript:void(0)">About</a></li>
+        </ul>
```

Comment:
- [ ] Hi for your mobile menu ill recommend giving it the highest z-index since it should appear in front of every other element
<img width="377" alt="image" src="https://user-images.githubusercontent.com/37901469/197232603-d3ac88ac-4864-4807-ab03-1fb2b5145845.png">